src/lib/lore.sh

```diff
@@ -35,16 +35,24 @@ declare -gA available_lore_mailing_lists
 declare -gr SEPARATOR_CHAR='Æ'
 
 # TODO: Remove hardcode and add setting for this
-# Number of patchsets needed to fetch when querying a lore mailing list
-declare -gr NUMBER_OF_PATCHSETS_PER_FETCH=30
+# Number of patchsets per page when querying a lore mailing list
+declare -g NUMBER_OF_PATCHSETS_PER_PAGE=30
```

Comment:
As part of this PR, I would like to see these configuration variables in the config file.